DirectProgramming/DPC++FPGA/Tutorials/Features/dsp_control/sample.json

```diff
@@ -0,0 +1,63 @@
+{
+  "guid": "77006CE8-2935-41C7-BA30-DF6EA07CD73A",
```

Comment:
Confirm you generated a unique GUID?

Reply:
Confirmed, this is a unique GUID I generated.